src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenApplicant2Approve.java

```diff
@@ -52,10 +58,16 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             return AboutToStartOrSubmitResponse.<CaseData, State>builder()
                 .data(data)
                 .errors(validationErrors)
-                .state(Draft)
+                .state(AwaitingApplicant2Response)
                 .build();
         }
 
+        data.setDueDate(LocalDate.now().plus(2, ChronoUnit.WEEKS));
+        data.setApplicant2ApprovedDueDate(LocalDate.now().plus(2, ChronoUnit.WEEKS));
```

Comment:
why do we have a separate field for this?

Reply:
@linusnorton I meant to only set the applicant2ApprovedDueDate. Is it better if I just reuse the dueDate field?

Reply:
yes please, I don't think we need `applicant2ApprovedDueDate`